.github/workflows/snapcraft.yml

```diff
@@ -0,0 +1,24 @@
+name: snapcraft
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download .snap artifact
+        uses: dsaltares/fetch-gh-release-asset@aa37ae5c44d3c9820bc12fe675e8670ecd93bd1c # 0.0.5
+        with:
+          repo: ipfs-shipyard/ipfs-desktop
+          version: tags/${{ github.event.release.tag_name }}
+          file: ipfs-desktop-${{ github.event.release.name }}-linux-amd64.snap
+
+      - name: Publish to Snapcraft
+        uses: snapcore/action-publish@f1879414dc5500e02a36f3d715bca6ddd438c913 # 1.0.2
+        with:
+          store_login: ${{ secrets.SNAP_STORE_LOGIN }}
```

Comment:
This secret was generated with instructions from  https://github.com/snapcore/action-publish#store-login